### DISCUSSION TITLE
增加对 OpenAI 类 API /models 接口的支持

### DISCUSSION THREAD

**Discussion by ZeroClover** [immersive-translate/immersive-translate#1882] [Ideas]
### 新功能 | New Feature

通过调用 /models 接口获取接口提供的模型列表以供选择。OpenAI 和 OpenAI API 兼容的服务大部分都支持这个接口，可以减少自行输入模型名称的麻烦。

### 为什么您想要添加这些功能 | Why you want to add these features

1. 无需手动维护模型列表

**Comment by theowenyoung** [comment]
目前已支持了哈。
